src/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -134,30 +137,34 @@ private void hideAll(Menu menu) {
      * @param toHide            List to save the options that must be shown in the menu.
      */
     private void filter(List<Integer> toShow, List <Integer> toHide) {
+
         boolean synchronizing = anyFileSynchronizing();
 
+        boolean videoPreviewing = anyFileVideoPreviewing();
```

Comment:
OK, they were not available either before, this was an inconsistency of multiple-selection mode. Works for me.